hc/front/tests/test_add_channel.py

```diff
@@ -37,5 +37,43 @@ def test_instructions_work(self):
             r = self.client.get(url)
             self.assertContains(r, "Integration Settings", status_code=200)
 
-    ### Test that the team access works
-    ### Test that bad kinds don't work
+    # Test that the team access works
+    def test_team_access_works(self):
+        form = {"kind": "email", "value": "bob@example.org"}
+        url = "/accounts/switch_team/%s/" % self.alice.username
+        self.client.login(username="bob@example.org", password="password")
+
+        """ Bob switches to Alice's team """
```

Comment:
comment this line instead of using doc strings

Reply:
This has been fixed